docs/docs/docs_es/basic-tutorial/performance.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Bonus: Rendimiento'
+---
+
+Nuestra implementación actual es perfectamente valida, pero hay algunos puntos importantes a considerar relacionados con el rendimiento. Desde que nuestra aplicación evolucionará desde ser un proyecto pequeño hasta convertirse en un programa corporativo de millones de líneas de código.
+
+Pensemos en que causará que cada uno de nuestro componentes se renderice:
+
+### `<TodoList />`
+
+Este componente esta suscrito a `filteredTodoListState`, que es un selector que tiene dependencia con `todoListState` y `todoListFilterState`. Esto significa que `TodoList` volverá a renderizar cuando los siguientes estados cambien:
+
+- `todoListState`
+- `todoListFilterState`
+
+### `<TodoItem />`
+
+Este componente esta suscrito a `todoListState`, entonces se volverá a renderizar cada vez que `todoListState` cambie, y también cuando el componente padre, `TodoList`, renderice.
+
+### `<TodoItemCreator />`
+
+Este componente no esta suscrito al estado de Recoil (`useSetRecoilState()` no crea alguna subscripción), por lo tanto, solo renderizará cuando su componente padre lo haga.
```

Comment:
está, alguna->ninguna, replace por lo tanto by y por ello and remove next comma, re-renderizará